modules/007-registrypackages/images/d8-ca-updater/scripts/d8-ca-updater

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# d8-ca-updater it's wrapper for update-ca-certificate/update-ca-trust commands, which use default way for introduce actual ca-certificate to system. 
+#
+# redhat and debian based distributive use different ways for install ca-certificates. 
+# redhat-based distros use /usr/share/pki/ca-trust-source/ca-bundle.trust.p11-kit file, which contain the complete Mozilla CA store. A system administrator can incorporate additional certificates by placing their PEM files under /etc/pki/ca-trust/source or /usr/share/pki/ca-trust-source directory and running 'update-ca-trust' command to set things straight. This will generate the /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem.
+# debian-based distros use /etc/ssl/certs/ca-certificates.crt, which is generated each time by 'update-ca-certificate' command. This command finds certificates in /usr/share/ca-certificates or /usr/local/share/ca-certificates directories and recreate  main ca-certificates.crt.
+
+set -x
+
+CERT_SRC_DIR="/opt/deckhouse/share/ca-certificates/mozilla/"
+mkdir -p /etc/ssl/certs
+
+detect_bundle() {
```

Comment:
We cannot discover the bundle by using the `update-ca-certificate` or `update-ca-trust` commands because these commands may not be present on the system. Without a discovered bundle, we do not know which directory to use for CA certificates.

Reply:
I think we should use another way:
```
root@dev-master-0:~# openssl version -d
OPENSSLDIR: "/usr/lib/ssl"
root@dev-master-0:~# ls -l /usr/lib/ssl
total 4
lrwxrwxrwx 1 root root   14 Mar 16  2022 certs -> /etc/ssl/certs
drwxr-xr-x 2 root root 4096 Aug 10  2022 misc
lrwxrwxrwx 1 root root   20 Jul  4  2022 openssl.cnf -> /etc/ssl/openssl.cnf
lrwxrwxrwx 1 root root   16 Mar 16  2022 private -> /etc/ssl/private
```

Reply:
And as i think i've refactor script to something like to:
```
if update-ca-trust -> add certificates && update
else if update-ca-certificates  -> add certificates && update
else
  find certs dir by openssl version -d, add certificates && update by d8-ca-updater itself.
```  
```